distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.admin.config.policy.AdminConfigPolicy-table-contents.adoc

```diff
@@ -0,0 +1,29 @@
+.[[org.codice.ddf.admin.config.policy.AdminConfigPolicy]]Admin Configuration Policy
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Feature and App Permissions
+|featurePolicies
+|String
+|When enabled,the desired features or apps will only be modifiable and viewable to users with the set attributes.
+The entry should be the format of: feature name/app name = "user attribute name=user attribute value"
```

Comment:
Should we maybe use some sort of formatting for the example? `feature name/app name = "user attribute name=user attribute value"` for example?